.changeset/silver-flowers-notice.md

```diff
@@ -0,0 +1,5 @@
+---
+"create-cloudflare": minor
```

Comment:
I debated with myself whether this is just a bug fix (i.e. patch) but I think we can go with it being a small feature.